Upstream-Status: Inappropriate [configuration]

Enable debugging for all builds
not building docs (need oxygen)

8/19/2010 - created by Qing He <qing.he@intel.com>
1/24/2011 - Disable Fedora/Debian checking/settings by Mark Hatle <mark.hatle@windriver.com>
3/15/2011 - Updated to newer version of sat-solver by Mark Hatle <mark.hatle@windriver.com>
3/16/2011 - Split original patch and simplify by Mark Hatle <mark.hatle@windriver.com>

diff -ur git.orig2/CMakeLists.txt git/CMakeLists.txt
--- git.orig2/CMakeLists.txt	2011-03-16 16:29:43.532032285 -0500
+++ git/CMakeLists.txt	2011-03-16 16:32:59.982034501 -0500
@@ -177,7 +177,7 @@
 
 MESSAGE(STATUS "Looking modules in ${CMAKE_MODULE_PATH}")
 
-set ( CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -Werror -Wall" )
+set ( CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -Werror -Wall -g" )
 set ( CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS} -g -O3" )
 set ( CMAKE_C_FLAGS_DEBUG     "${CMAKE_C_FLAGS} -g3 -O0" )
 
@@ -186,7 +186,7 @@
 ADD_SUBDIRECTORY(tools)
 ADD_SUBDIRECTORY(tests)
 ADD_SUBDIRECTORY(examples)
-ADD_SUBDIRECTORY(doc)
+#ADD_SUBDIRECTORY(doc)
 
 MESSAGE(STATUS "version: ${VERSION}")
 
